lib/taskana-core/src/main/java/pro/taskana/configuration/TaskanaEngineConfiguration.java

```diff
@@ -73,6 +74,9 @@
     // List of configured classification types
     protected List<String> classificationTypes = new ArrayList<String>();
 
+    // List of configured classification categories
+    protected List<String> classificationCategories = new ArrayList<String>();
+

```

Comment:
classificationTypes and classificationCategories should better be Set s to prevent duplicate entries...